lib/SIL/SILPrinter.cpp

```diff
@@ -3026,6 +3057,72 @@ void SILDefaultWitnessTable::dump() const {
   print(llvm::errs());
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+void SILDifferentiabilityWitness::print(
+    llvm::raw_ostream &OS, bool verbose) const {
+  OS << "// differentiability witness for "
+     << demangleSymbol(originalFunction->getName()) << "\n";
```

Comment:
Use a character literal when outputting one character. 
```suggestion
     << demangleSymbol(originalFunction->getName()) << '\n';
```

Reply:
Done in https://github.com/apple/swift/pull/27487/commits/2073f863826fb305c1fc105a520d84de1e1463ec.